pilot/pkg/util/sets/string.go

```diff
@@ -139,3 +139,18 @@ func (s Set) Equals(other Set) bool {
 func (s Set) Empty() bool {
 	return len(s) == 0
 }
+
+// Diff takes a pair of Sets, and returns the elements that occur only on the left and right set.
+func (s Set) Diff(other Set) (left []string, right []string) {
```

Comment:
Looks like this is not being used any more? But it is OK to keep it for future use